communities.json

```diff
@@ -771,6 +771,13 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "ballista.png",
+      "title": "Ballista Distributed Compute",
+      "quote": "Join our Discord channel to chat with the core contributors.",
+      "inviteCode": "ballista",
```

Comment:
It looks like this invite code doesn't currently exist. I cannot identify your server without a valid invite code.